.github/workflows/auto-pr-upstream.yaml

```diff
@@ -0,0 +1,42 @@
+name: Auto PR from Upstream
+
+on:
+  schedule:
+    - cron: '0 0 * * Wed' # Every Wednesday at midnight
```

Comment:
Can we have this run at 10am ET on Tuesday so the PR is something we can look at for sprint planning?

Reply:
I'm having it run at 11:59 PM Monday, so that we have some time before sprint planning to look at the PR if anything goes wrong.